pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBase.java

```diff
@@ -26,17 +27,28 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
 import java.util.function.Supplier;
 import org.apache.pulsar.client.admin.PulsarAdmin;
 import org.apache.pulsar.client.admin.PulsarAdminException;
 import org.apache.pulsar.client.admin.PulsarAdminException.ConnectException;
+import org.apache.pulsar.client.admin.internal.PulsarAdminImpl;
 
 public abstract class CmdBase {
     protected final JCommander jcommander;
     private final Supplier<PulsarAdmin> adminSupplier;
     private PulsarAdmin admin;
     private IUsageFormatter usageFormatter;
 
+    /**
+     * Default read timeout in milliseconds.
+     * Used if not found from configuration data in {@link #getReadTimeoutMs()}
+     */
```

Comment:
@JooHyukKim this PR is labeled with `doc-required`, and explanations have been added to API code files. Does it mean that the doc has been already added?

Reply:
OK